e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/about-goods-or-services/save-and-back.spec.js

```diff
@@ -0,0 +1,109 @@
+import { submitButton, saveAndBackButton } from '../../../../pages/shared';
+import { aboutGoodsOrServicesPage } from '../../../../pages/insurance/policy-and-export';
+import partials from '../../../../partials';
+import { TASKS } from '../../../../../../content-strings';
+import { ROUTES, FIELD_IDS, FIELD_VALUES } from '../../../../../../constants';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+import application from '../../../../../fixtures/application';
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      ABOUT_GOODS_OR_SERVICES: { DESCRIPTION },
+    },
+  },
+} = FIELD_IDS;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.policyTypeAndExports;
+
+context('Insurance - Policy and exports - About goods or services - Save and go back', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    task.link().click();
+
+    cy.completeAndSubmitPolicyTypeForm(FIELD_VALUES.POLICY_TYPE.SINGLE);
+
+    cy.completeAndSubmitSingleContractPolicyForm();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const expected = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES}`;
+      cy.url().should('eq', expected);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  describe('when submitting an empty form via `save and go back` button', () => {
+    it(`should redirect to ${ROUTES.INSURANCE.ALL_SECTIONS}`, () => {
```

Comment:
a dereference for all occurrences please :P we seem to be using ROUTES.INSURANCE  a few times